.eslintignore

```diff
@@ -6,6 +6,10 @@ e2e-tests/**
 examples/with-eslint/**
 examples/with-typescript-eslint-jest/**
 examples/with-kea/**
+examples/with-custom-babel-config/**
+examples/with-flow/**
+examples/with-mobx-state-tree/**
+examples/with-mobx/**
```

Comment:
ESLint is erroring any examples with experimental syntax. They're not picking up their local `.babelrc` files for some reason and don't want to inherently change user-facing examples (hence why I threw them in `.eslintignore` for now).

TODO: I still need to add `eslint-config-next` to all the examples and ensure they work with all the latest features